### PR TITLE
CDAP-8506 Mention spark and workers in REST Logging docs

### PR DESCRIPTION
**Pull request by Denton-L**
JIRA: https://issues.cask.co/browse/CDAP-8506
Bamboo: http://builds.cask.co/browse/CDAP-DQB266

[This](http://builds.cask.co/artifact/CDAP-DQB266/shared/build-1/Docs-HTML/4.1.0-SNAPSHOT/en/reference-manual/http-restful-api/logging.html) is the page of interest.

### CODE REVIEW

File: cdap-docs/reference-manual/source/http-restful-api/logging.rst [hunk 1]
```diff
@@ -41,7 +41,7 @@ downloaded with the Logging HTTP RESTful API. To do that, send an HTTP GET reque
    * - ``app-id``
      - Name of the application being called
    * - ``program-type``
-     - One of ``flows``, ``mapreduce``, ``services``, ``spark``, or ``workflows``
+     - One of ``flows``, ``mapreduce``, ``services``, ``spark``, ``workers``, or ``workflows``
    * - ``program-id``
      - Name of the program (*flow*, *MapReduce*, *service*, *Spark*, *workflow*) being called
```
Comment (awholegunch) [inline cdap-docs/reference-manual/source/http-restful-api/logging.rst]: then this line should have ``*worker*`` added.

File: cdap-docs/reference-manual/source/http-restful-api/logging.rst [hunk 2]
```diff
@@ -79,7 +79,7 @@ To download logs for a program run, send an HTTP GET request::
...
```
Comment (awholegunch) [inline cdap-docs/reference-manual/source/http-restful-api/logging.rst]: ditto

File: cdap-docs/reference-manual/source/http-restful-api/logging.rst (hunk 1 above)
Comment (awholegunch) [reply cdap-docs/reference-manual/source/http-restful-api/logging.rst]: But as this list keeps getting longer, why don't we change these to:
```
Name of the program being called
```
as that is much simpler

File: cdap-docs/reference-manual/source/http-restful-api/logging.rst (hunk 2 above)
Comment (awholegunch) [inline cdap-docs/reference-manual/source/http-restful-api/logging.rst]: Name of the program being called

**Review by awholegunch — CHANGES_REQUESTED** [review]

**Comment by Denton-L** [comment]
@awholegunch, please take another look.

**Review by awholegunch — APPROVED** [review]

**Pull request merged**